packages/gatsby/src/services/initialize.ts

```diff
@@ -85,6 +85,8 @@ export async function initialize({
   }
   if (!args) {
     reporter.panic(`Missing program args`)
+    // @ts-ignore - early return so that args is defined
```

Comment:
Since `IGatsbyState` has more "possibly undefined" keys now it complains about that on `args` further down the line. Doing an early return makes TS happy

Reply:
`reporter.panic` _should_ indicate that nothing can execute after that with return type being `never` ( https://github.com/gatsbyjs/gatsby/blob/94b587577b2d42f415d67f8be78c9269823f944b/packages/gatsby-cli/src/reporter/reporter.ts#L83-L90 )

It also didn't fail on those bits before - so it's confusing why it sudennly need it here?

Reply:
Mh, removing it again it seems to work now. Maybe something other I fixed made this obsolete now. Will revert